src/main/resources/naver.yml

```
@@ -13,4 +13,7 @@ spring.config.activate.on-profile: local
 spring.config.activate.on-profile: dev
 
 ---
-spring.config.activate.on-profile: prod
\ No newline at end of file
+spring.config.activate.on-profile: prod
+
+---
+spring.config.activate.on-profile: test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)